app/components/WriteUs.vue

```diff
@@ -24,6 +24,13 @@ import appAvailability from 'nativescript-appavailability'
 import config from '../config.js'
 
 export default {
+  data: () => {
+    return {
+      backgroundColor: config.colors.appBackgroundColor,
```

Comment:
We could acquire some code reading quality if we use object destructuring to the config colors attributes.